rq/job.py

```diff
@@ -493,6 +493,16 @@ def save(self, pipeline=None, include_meta=True):
         connection.hmset(key, self.to_dict(include_meta=include_meta))
         self.cleanup(self.ttl, pipeline=connection)
 
+    def update_meta(self):
+        """Stores job meta from the job instance to the corresponding Redis key."""
+        key = self.key
+        meta = dumps(self.meta) if self.meta else None
+        pipeline = self.connection._pipeline()
+        pipeline.hdel(key, 'meta')
```

Comment:
@katichev please update this PR sometime in the next few days. I plan on releasing 0.8.0 to PyPI soon.